packages/epics/src/membership/components/members/member-tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react';
+import { Tabs, TabsList, TabsTrigger } from '@hypha-platform/ui/server';
+
+type MemberTabsProps = {
+  activeTab: string;
+  setActiveTab: (tab: string) => void;
+};
+
+export const MemberTabs: FC<MemberTabsProps> = ({
+  activeTab,
+  setActiveTab,
+}) => {
+  return (
+    <Tabs
+      value={activeTab}
+      onValueChange={setActiveTab}
+      className="w-full mt-3 mb-6"
+    >
+      <TabsList>
+        <TabsTrigger value="all" variant="outlined">
+          All
+        </TabsTrigger>
+        <TabsTrigger value="active" variant="outlined">
+          Active
+        </TabsTrigger>
+        <TabsTrigger value="inactive" variant="outlined">
+          Inactive
+        </TabsTrigger>
+        <TabsTrigger value="applicant" variant="outlined">
+          Applicants
+        </TabsTrigger>
+        <TabsTrigger value="rejected" variant="outlined">
+          Rejected
+        </TabsTrigger>
+      </TabsList>
+    </Tabs>
+  );
+};
```

Comment:
Please let's try to extract `data` from `markup`. This makes maintanence and integration easier and isn't adding additional work when done continiously.

```suggestion
const tabs = [
  { label: 'All', value: 'all' },
  { label: 'Active', value: 'active' },
  { label: 'Inactive', value: 'inactive' },
  { label: 'Applicants', value: 'applicant' },
  { label: 'Rejected', value: 'rejected' },
];

export const MemberTabs: FC<MemberTabsProps> = ({
  activeTab,
  setActiveTab,
}) => {
  return (
    <Tabs
      value={activeTab}
      onValueChange={setActiveTab}
      className="w-full mt-3 mb-6"
    >
      <TabsList>
        {tabs.map((tab) => (
          <TabsTrigger key={tab.value} value={tab.value} variant="outlined">
            {tab.label}
          </TabsTrigger>
        ))}
      </TabsList>
    </Tabs>
  );
};
```

Reply:
By doing so, you'll notice similarities to extract shared components.